doc/repository-spec-v2.md

```diff
@@ -64,6 +64,9 @@ repository.
 
 ### Inspect a specific version of a package
```

Comment:
Let's add `(deprecated)` to the section title

Reply:
Done